### PR TITLE
Fix "Cannot read property 'number' of undefined" for PullRequest

### PR DESCRIPTION
**Pull request by kongo555**

### CODE REVIEW

**Comment by dbaileychess** [comment]
I ran into this issue as well, it would be a good fix.

**Comment by MoonScript** [comment]
@jenschelkopf can we please get this merged?  I'd love to be able to use this on pull requests as well.

File: dist/index.js [hunk 1]
```diff
@@ -403,7 +403,8 @@ const { correctRecipients, correctMessage } = __webpack_require__(278);
 
 async function run() {
   try {
-    const issueNumber = github.context.payload.issue.number;
+    const payload = github.context.payload;
+    const issueNumber = (payload.issue || payload.pull_request || payload).number;
```
Comment (MoonScript) [inline dist/index.js:407]: @kongo555 does this same fix need to be applied to the `index.js` file in the root folder?
https://github.com/jenschelkopf/issue-label-notification-action/blob/master/index.js#L7

**Review by MoonScript — APPROVED** [review]
+1 unless this change also should be applied to the `index.js` file in the root folder (line 7).

**Pull request closed without merging**